src/content/changelog/4-77-2020-10-05-2020.md

```diff
@@ -0,0 +1,31 @@
+---
+title: '4.77.0: 2020-10-05'
+date: 2020-10-05T04:00:00.000Z
+version: 4.77.0
+changelog:
+  - API
+---
+
+### Added
+
+- Added the `messages` field to the Event schema object. The message field provides additional information about the event. Additional information may include, but is not limited to, a more detailed representation of events which can help diagnose non-obvious failures. This new field is available on the following Account Event endpoints:
+
+    - Event List ([GET /account/events](/api/v4/account-events))
+    - Event View ([GET /account/events/{eventId}](/api/v4/account-events-event-id))
+
+- Added the `deprecated` field to the Kernel schema object. If this Kernel is marked as deprecated, this field has a value of true; otherwise, this field is false. This new field is available on the Linode Instances Kernel endpoints:
+
+    - Kernels List ([GET /linode/kernels](/api/v4/linode-kernels))
+    - Kernel View ([GET /linode/kernels/{kernelId}](/api/v4/linode-kernels-kernel-id))
+
+### Changed
+
+- Updated the endpoint names to follow the convention `noun` followed by `verb`. For example, `View Account` is now named `Account View`.
+
+- Updated the description for the Object Storage Key Create ([POST /object-storage/keys](/api/v4/object-storage-keys/#post)) endpoint to include options of key creation with or without using the `bucket_access` array field.
```

Comment:
This line reads a little confusing to me. One suggestion:
" Updated the description for the Object Storage Key Create endpoint with information on the available Access Key and Limited Access Key creation options."

Reply:
thank you! updated